x/slashing/keeper/keeper.go

```diff
@@ -176,34 +117,6 @@ func (k Keeper) setBlockTimes(batch db.Batch, blockTimes []time.Time) {
 	batch.Set([]byte(dbKeyBlockTimes), bz.Bytes())
 }
 
-func (k Keeper) slashValidator(ctx sdk.Context, batch db.Batch, consAddr sdk.ConsAddress, infractionHeight int64, power int64, slashFactor sdk.Dec) {
-	k.sk.Slash(ctx, consAddr, infractionHeight, power, slashFactor)
-
-	// Mint the slashed coins and assign them to the distribution pool.
-	slashAmount := calculateSlashingAmount(power, slashFactor)
-	k.Logger(ctx).Info("calculating slash amount",
-		"power", power, "factor", slashFactor, "amount", slashAmount.String(),
-		"tokensFPower", sdk.TokensFromConsensusPower(power).String(),
-		"calc", sdk.TokensFromConsensusPower(power).ToDec().Mul(slashFactor).String())
-	stakingDenom := k.sk.BondDenom(ctx)
-	coins := sdk.NewCoins(sdk.NewCoin(stakingDenom, slashAmount))
-	err := k.bankKeeper.MintCoins(ctx, types.ModuleName, coins)
-	if err != nil {
-		panic(err)
-	}
-
-	k.Logger(ctx).Info("transfer to penalty account", "amounts", coins)
-	err = k.bankKeeper.SendCoinsFromModuleToModule(ctx, types.ModuleName, types.PenaltyAccount, coins)
-	if err != nil {
-		panic(err)
-	}
-
-	activePenalties := k.getPendingPenalties()
-	activePenalties.Add(consAddr.String(), coins)
-	k.setPendingPenalties(batch, activePenalties)
-	k.Logger(ctx).Info("set penalties", "active", activePenalties)
-}
-
 // Adopted from cosmos-sdk/x/staking/keeper/slash.go
 func calculateSlashingAmount(power int64, slashFactor sdk.Dec) sdk.Int {
```

Comment:
Does not appear to be used anymore after the removal of slashValidator.

Reply:
Removed.